src/components/MovieCard/MovieCard.tsx

```diff
@@ -2,6 +2,51 @@ import React from 'react';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
-);
+interface Props {
+  title: string;
+  description: string;
+  imdbUrl: string;
+  imgUrl: string;
+}
+
+export const MovieCard: React.FC<Props> = ({
+  title,
+  description,
+  imdbUrl,
+  imgUrl,
+}) => {
+  return (
+    <>
```

Comment:
right,
- use implicit return
- remove redundant fragment

```suggestion
}) => (
```